### PR TITLE
fix(create-bootable-usb.sh) confirmation prompt

### PR DESCRIPTION
**Pull request by lennartpj**
## Type of Change
- [ ] New feature
- [x] Bug fix
- [ ] Documentation update
- [ ] Refactoring
- [ ] Hotfix
- [ ] Security patch
- [ ] UI/UX improvement

## Description
Fixing the confirmation prompt by converting everything to lower case and allowing yes and y as answers.

## Testing
tested this and works as expected

## Impact
minor

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

**Review by adamperkowski — APPROVED** [review]

File: core/tabs/utils/create-bootable-usb.sh [hunk 1]
```diff
@@ -233,7 +233,7 @@ write_iso(){
     printf "%b" "${RED}WARNING: This will erase all data on ${USB_DEVICE}. Are you sure you want to continue? (y/N): ${RC}"
     read -r CONFIRMATION
 
-    if [ "$CONFIRMATION" != "yes" ]; then
+    if [ "$(echo "$CONFIRMATION" | tr '[:upper:]' '[:lower:]')" != "yes" ] && [ "$(echo "$CONFIRMATION" | tr '[:upper:]' '[:lower:]')" != "y" ]; then
```
Comment (lj3954) [inline core/tabs/utils/create-bootable-usb.sh:236]:
```suggestion
    CONFIRMATION="$(echo "$CONFIRMATION" | tr '[:upper:]' '[:lower:]')"
    if [  "${CONFIRMATION}" != "yes" ] && [ "${CONFIRMATION}" != "y" ]; then
```

**Review by lj3954 — CHANGES_REQUESTED** [review]

File: core/tabs/utils/create-bootable-usb.sh (hunk 1 above)
Comment (lennartpj) [reply core/tabs/utils/create-bootable-usb.sh:236]: wow, that was fast!
I gave it a quick test and it seems to work. 
thank you!

**Comment by adamperkowski** [comment]
The CI failing here is unrelated to the changes.

**Comment by cartercanedy** [comment]
> The CI failing here is unrelated to the changes.

Hmm...

Looks like he needs to put in some shellcheck directives to ignore the warnings

https://github.com/ChrisTitusTech/linutil/blob/e96b75a890a149fe9af83bbd1040f15590708fa9/core/tabs/utils/create-bootable-usb.sh#L17

https://github.com/ChrisTitusTech/linutil/blob/e96b75a890a149fe9af83bbd1040f15590708fa9/core/tabs/utils/create-bootable-usb.sh#L23

https://github.com/ChrisTitusTech/linutil/blob/e96b75a890a149fe9af83bbd1040f15590708fa9/core/tabs/utils/create-bootable-usb.sh#L25

**Review by cartercanedy — CHANGES_REQUESTED** [review]
```sh
    # shellcheck disable=SC2086
```

Apply shellcheck directives like this to the rest of the unquoted `${DEPENDENCIES}` instances

**Review by lj3954 — APPROVED** [review]

**Review by jeevithakannan2 — APPROVED** [review]

**Pull request merged**